src/locales/es.json

```diff
@@ -25,76 +25,76 @@
   "Loading": "Cargando",
   "Login": "Iniciar sesión",
-  "Logging in": "Logging in",
-  "Logging out": "Logging out",
+  "Logging in": "Iniciando sesión",
+  "Logging out": "Cerrando sesión",
   "Logout": "Cerrar sesión",
   "Mismatch": "Desajuste",
   "More": "Más",
-  "New notification received.": "New notification received.",
-  "No items found": "No items found",
+  "New notification received.": "Nueva notificación recibida.",
+  "No items found": "No se encontraron artículos",
   "No inventory details found": "No se encontraron detalles de inventario",
   "Not in stock": "No disponible en el inventario",
   "No products found": "No se encontraron productos",
   "No reason": "Sin razón",
-  "No notifications to show": "No notifications to show",
-  "No picker assigned.": "No picker assigned.",
+  "No notifications to show": "No hay notificaciones para mostrar",
+  "No picker assigned.": "No hay recolector asignado.",
   "No picker found": "No se encontró recolector",
-  "No records found.": "No records found.",
+  "No records found.": "No se encontraron registros.",
   "No time zone found": "No se encontró zona horaria",
-  "Notifications": "Notifications",
-  "Notification Preference": "Notification Preference",
-  "Notification preferences not found.": "Notification preferences not found.",
-  "Notification preferences updated.": "Notification preferences updated.",
-  "Notification preferences not updated. Please try again.": "Notification preferences not updated. Please try again.",
+  "Notifications": "Notificaciones",
+  "Notification Preference": "Preferencias de notificación",
+  "Notification preferences not found.": "Preferencias de notificación no encontradas.",
+  "Notification preferences updated.": "Preferencias de notificación actualizadas.",
+  "Notification preferences not updated. Please try again.": "Preferencias de notificación no actualizadas. Inténtalo de nuevo.",
   "Open": "Abierto",
   "OMS": "OMS",
   "OMS instance": "Instancia de OMS",
-  "On rejecting this item, will be sent an email with alternate fulfillment options and this order item will be removed from your dashboard.": "On rejecting this item, { customerName } will be sent an email with alternate fulfillment options and this order item will be removed from your dashboard.",
-  "On rejecting this order, will be sent an email with alternate fulfilment options and this order will be removed from your dashboard.": "On rejecting this order, { customerName } will be sent an email with alternate fulfilment options and this order will be removed from your dashboard.",
+  "On rejecting this item, will be sent an email with alternate fulfillment options and this order item will be removed from your dashboard.": "Al rechazar este artículo, se enviará a {customerName} un correo electrónico con opciones de cumplimiento alternativas y este artículo del pedido se eliminará de su panel de control..",
```

Comment:
Extra dot "." at the end of the line.
Also in following: [https://github.com/hotwax/bopis/pull/367/files#diff-62af77f19ef0f45dcc11f8df29738b97e55b0878c7bb495e356e921e22738aa2R93-R95](url)